exchanges/oex/oex_test.go

```diff
@@ -0,0 +1,250 @@
+package oex
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/config"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+)
+
+// Please supply your own keys here for due diligence testing
+const (
+	testAPIKey              = ""
+	testAPISecret           = ""
+	canManipulateRealOrders = false
+)
+
+var o Oex
+var setupRan bool
+var m sync.Mutex
+
+func TestSetup(t *testing.T) {
+	t.Parallel()
+	m.Lock()
```

Comment:
If you use a sync.Mutex, you can parallelise the below tests for faster testing otherwise it's not required as the tests will run sequentially by default